rts/lra-examples/run.sh

```diff
@@ -72,12 +77,12 @@ function start_and_test_service {
 
   if [ "$1" = "true" ]; then
     echo " ===== waiting for recovery ......."
-    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery 
+    curl ${CURL_IP_OPTS} http://localhost:${coord_port}/lra-recovery-coordinator/recovery
     # sometimes it can take two scans to complete recovery
-    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery 
+    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery
```

Comment:
Is IPV4 not required here?

Reply:
This curl is sending an HTTP GET request to a JAX-RS endpoint which will be running inside a container that accepts IPv4 requests.

Reply:
+1, the other PR changes all use ${CURL_IP_OPTS} in the curl request.

Reply:
thanks, I overlooked this